iocore/hostdb/HostDB.cc

```diff
@@ -439,37 +327,24 @@ HostDBCache::start(int flags)
     Warning("Please set 'proxy.config.hostdb.storage_path' or 'proxy.config.local_state_dir'");
   }
 
-  hostDBStore = new Store;
-  hostDBSpan = new Span;
-  hostDBSpan->init(storage_path, storage_size);
-  hostDBStore->add(hostDBSpan);
-
-  Debug("hostdb", "Opening %s, size=%d", hostdb_filename, hostdb_size);
-  if (open(hostDBStore, "hostdb.config", hostdb_filename, hostdb_size, reconfigure, fix, false /* slient */) < 0) {
-    ats_scoped_str rundir(RecConfigReadRuntimeDir());
-    ats_scoped_str config(Layout::relative_to(rundir, "hostdb.config"));
+  // Combine the path and name
+  char full_path[2 * PATH_NAME_MAX];
```

Comment:
Yea, TBH I'm not sure why thats like that-- it was before (in a different part of the code-- moved to here. Presumably we actually know what the size is-- since we have the filename and path... I can definitely clean that :)